substrate/frame/conviction-voting/src/tests.rs

```diff
@@ -193,7 +194,7 @@ impl Config for Test {
 pub fn new_test_ext() -> sp_io::TestExternalities {
 	let mut t = frame_system::GenesisConfig::<Test>::default().build_storage().unwrap();
 	pallet_balances::GenesisConfig::<Test> {
-		balances: vec![(1, 10), (2, 20), (3, 30), (4, 40), (5, 50), (6, 60)],
+		balances: vec![(1, 11), (2, 21), (3, 31), (4, 41), (5, 51), (6, 61)],
```

Comment:
Why tweak all these?

Reply:
absolutely 👍

Reply:
@liamaharon revamped to using freezes now. Still have to update the naming though.